roles/cs.aws-rds/tasks/main.yml

```diff
@@ -6,13 +6,32 @@
     description: "{{ aws_rds_subnet_group_name }}"
     subnets: "{{ aws_rds_subnet_group_subnets }}"
 
+- name: Create RDS parameter group
+  rds_param_group:
+    state: present
+    region: "{{ aws_region }}"
+    name: "{{ aws_rds_param_group_name }}"
+    description: "{{ aws_rds_param_group_desc }}"
+    engine: "{{ aws_rds_param_group_engine }}"
+    tags: "{{ aws_tags_default }}"
+    params: 
```

Comment:
can you add "extra params" variable to allow extending that in project configuration?

Reply:
Done